fusesoc/utils.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, cmd, args=[], cwd=None):
     def run(self):
         """Runs the cmd with args after converting them all to strings via str
         """
-        logger.debug(self.cwd)
+        logger.debug(self.cwd or "./")
```

Comment:
Was printing `Debug: None` when cwd was not set, can remove from the PR if this isn't desirable